cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -116,7 +116,7 @@ func TestAddFlags(t *testing.T) {
 		"--max-endpoints-per-slice=200",
 		"--min-resync-period=8h",
 		"--namespace-sync-period=10m",
-		"--node-cidr-mask-size=48",
+		"--node-cidr-mask-size=48,98",
```

Comment:
Would a better format be to have the string be represented as something like "v4:48,v6:98", so the order does not matter or we do want to use implicit positional interpretation?

Reply:
Switched to using `--node-cidr-mask-size-ipv4` and `--node-cidr-mask-size-ipv6` flags explicitly based on the discussion on this PR and sig-network call.